src/main/java/com/duke/Duke.java

```diff
@@ -0,0 +1,197 @@
+package com.duke;
+
+import com.file.FileManager;
+import com.task.Deadline;
+import com.task.Event;
+import com.task.Task;
+import com.task.Todo;
+
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.io.IOException;
+
+public class Duke {
+
+    public static FileManager fileTask;
+    static ArrayList<Task> list = new ArrayList<>();
+    public static String[] dukeLines = new String[100];
+    public static int dukeLineCount = 0;
+    public final static String LINESEPARATER =  "\t______________________________________________________________________";
+
+    public static void addTask(Task item) {
+        list.add(item);
+    }
+    private boolean active = false;
+
+    public Duke(){
+        active = true;
+        printLine();
+        System.out.println("\tHello from");
+        System.out.println("\t ____        _        ");
+        System.out.println("\t| _ \\ _   _| | _____ ");
+        System.out.println("\t| | | | | | | |/ / _ \\");
+        System.out.println("\t| |_| | |_| |   <  __/");
+        System.out.println("\t|____/ \\__,_|_|\\_\\___|");
+        loadFromFile();
+        endLine();
+    }
+
+
+    public void loadFromFile(){
+        try {
+            fileTask = new FileManager("Task.txt");
+            ArrayList<String> Lines = fileTask.readFile();
+            for(String line : Lines) {
+                if(line.contains("[T]")) {
+                    String description = line.substring(6);
+                    list.add(new Todo(description));
+                }
+                else if (line.contains("[E]")) {
+                    String description = line.substring(6, line.indexOf("("));
+                    String ddl = line.substring(line.indexOf("(")+1,line.indexOf(")"));
+                    list.add(new Event(description,ddl));
+                }else if (line.contains("[D]")) {
+                    String description = line.substring(6, line.indexOf("("));
+                    String ddl = line.substring(line.indexOf("(")+1,line.indexOf(")"));
+                    list.add(new Deadline(description,ddl));
+                }
+                list.get(list.size()-1).setDone(line.contains("[X]"));
+            }
```

Comment:
The if-else statement layout here deviates from the [standard layout](https://se-education.org/guides/conventions/java/basic.html#layout) - E.g. space before `if` condition and missing `else` block.